### PR TITLE
{bio} [dummy]  MuTect-1.1.4-Java-1.7.0_76 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/86f75990071c3d368a7c for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/f05cde3df4d63502a772 for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/6577fa26983a086a3dcf for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/56a2e4f2633124dfa5b8 for a full test report.

**Comment by boegel** [comment]
@pescobar: please fix the directory name: `MuTect` vs `muTect`

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/85b73d9a6437e0f92f70 for a full test report.

**Comment by boegel** [comment]
@pescobar: `ERROR Sanity check failed: no file of ('MuTect-1.1.4.jar',) in ...`

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ and accurate identification of somatic point mutations in next generation sequencing
+ data of cancer genomes. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://www.broadinstitute.org/cancer/cga/sites/default/files/data/tools/mutect/']
+sources = ['muTect-%(version)s-bin.zip']
+
+java = 'Java'
+javaver = '1.7.0_76'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb]: I would have really liked it to standardize in the bio world upon `1.7.0_80`, but I leave that up to you...
this looks good overall, fingers crossed it works for many.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6898637d45d13b72ad3a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb]: Well, it does have the versionsuffix... the `1.7.0_80` version is only a PR away ;)

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+name = 'MuTect'
+version = '1.1.4'
+
+homepage = 'http://www.broadinstitute.org/cancer/cga/mutect'
+description = """ MuTect is a method developed at the Broad Institute for the reliable
+ and accurate identification of somatic point mutations in next generation sequencing
+ data of cancer genomes. """
+
+toolchain = {'name': 'dummy', 'version': ''}
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb]: `dummy/?`

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+dependencies = [(java, javaver)]
+
+sanity_check_paths = {
+    'files': ["muTect-%(version)s.jar"],
+    'dirs': [],
+}
+
+modloadmsg = """
+to execute: java -jar \\$EBROOTMUTECT/muTect-%(version)s.jar
+"""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb]: one-liners are sweeeet :)

File: easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/m/MuTect/MuTect-1.1.4-Java-1.7.0_76.eb]: deps are involved, so version must be empty

also: this is merged already :)

**Pull request merged**